src/stack_pr/cli.py

```diff
@@ -559,6 +549,13 @@ def add_or_update_metadata(e: StackEntry, needs_rebase: bool, verbose: bool) ->
     return True
 
 
+def fix_branch_name_template(branch_name_template: str):
+    if "$ID" not in branch_name_template:
+        return f"{branch_name_template}/$ID"
```

Comment:
Optional: add a warning when we fix this? I could see this tripping someone up if they skim the documentation.

Reply:
I didn't put any warning/error here because I don't want to impose a change for anyone who has used this feature before. IMO nothing should break (because of this code) if `$ID` is not present, so there is no real need in warning the user.